semcore/data-table/src/Body.tsx

```diff
@@ -325,6 +326,11 @@ class Body extends Component<AsProps, State> {
       return <SBodyWrapper>{body}</SBodyWrapper>;
     }
 
+    const scrollContainerRefs = [$scrollRef, this.scrollContainerRef];
```

Comment:
Are you sure that we haven't broken the backward capability as ref might be used to access body but not scroll container?

Reply:
Could you please add scrollRef instead?

Reply:
added